docs/support_files/api_reference.md

```diff
@@ -132,6 +133,37 @@ Set the default timeout for asynchronous steps. Defaults to `5000` milliseconds.
 
 ---
 
+#### `setDefinitionFunctionWrapper(wrapper)`
```

Comment:
I think it would make sense to rework the content and example here. The screenshot use case is much better served by `AfterStep` now. Also we should probably call out that it's deprecated?